spec/lib/shoryuken/middleware/server/non_retryable_exception_spec.rb

```
@@ -0,0 +1,214 @@
+      expect {
+        subject.call(TestWorker.new, queue, sqs_msg, sqs_msg.body) { raise StandardError, 'test error' }
+      }.to raise_error(StandardError, 'test error')
+    end
+  end
+
+  context 'when exception is in non_retryable_exceptions list' do
+    it 'deletes the message and does not re-raise' do
+      TestWorker.get_shoryuken_options['non_retryable_exceptions'] = [StandardError]
+
+      expect(sqs_queue).to receive(:delete_messages).with(entries: [
+                                                           { id: '0', receipt_handle: sqs_msg.receipt_handle }
```

RuboCop (1.84.0)

[convention] Use 2 spaces for indentation in an array, relative to the first position after the preceding left parenthesis.

(Layout/FirstArrayElementIndentation)